README.md

```diff
@@ -93,13 +93,31 @@ Type: `function`
 
 Filter out files. [more details](#hooks-options)
 
+**Use-case**:  Omit [dll-chunks] ([issue](https://github.com/danethurber/webpack-manifest-plugin/issues/46)):
```

Comment:
It's not specific to dll chunks, also dynamic one (when you use `import(...).then(...)`)